action/has_vm_test.go

```diff
@@ -50,8 +50,7 @@ var _ = Describe("HasVM", func() {
 				vmFinder.FindErr = errors.New("fake-find-err")
 
 				found, err := action.Run(1234)
-				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("fake-find-err"))
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
CPI should allow VMNotFoundError otherwise it will block the deployment. 
The action is compliance with the specification of CPI API v1
http://bosh.io/docs/cpi-api-v1.html